README.md

```diff
@@ -1,9 +1,13 @@
 
 <p align="center">
-<img src="docs/logo/logo_1024.png" width="500">
+<img src="https://github.com/bancaditalia/black-it/blob/3342fd725e2273d6cdf92fb2d5fc1565993c7500/docs/logo/logo_1024.png?raw=true" width="500">
```

Comment:
1. the use of a query string parameter (`?raw=True`) may be bad for HTTP caching.
2. visiting that url (for example via `curl --verbose <URL>`) gives an HTTP/302 that points back to the actual content, stored at https://github.com/bancaditalia/black-it/raw/3342fd725e2273d6cdf92fb2d5fc1565993c7500/docs/logo/logo_1024.png
3. this is one of the few cases for which I'd rather use a link to the branch instead of a specific commit

I'd try to use https://raw.githubusercontent.com/bancaditalia/black-it/main/docs/logo/logo_1024.png here.

Is that ok?

Reply:
Yes of course, thanks

Reply:
Replaced in **v2**.